lib/hanami/entity.rb

```diff
@@ -219,9 +219,11 @@ def ==(other)
     #
     #   user = User.new(id: 23, name: 'Luca')
     #   user.to_h # => { :id => 23, :name => "Luca" }
+    #   user.to_hash # => { :id => 23, :name => "Luca" }
     def to_h
       Hash[attribute_names.map { |a| [a, read_attribute(a)] }]
```

Comment:
@joneslee85 This won't make `#to_h` compatible with the rest of Ruby ecosystem, which doesn't know about our `Utils::Hash`.